<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ver_cortez</class>
 <widget class="QMainWindow" name="ver_cortez">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>869</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: rgb(249, 247, 245)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Cortes de imagenes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="corte_sagital">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imagenes/img proy inf/truncfinal01.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="corte_coronal">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imagenes/img proy inf/truncfinal01.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="corte_axial">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imagenes/img proy inf/truncfinal01.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="normalizar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;; background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>Normalizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filtrar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>630</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;;  background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>Filtrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="umbralizacin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>680</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;; background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>Umbralización</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atras">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>750</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;; background-color: rgb(159, 183, 190)</string>
    </property>
    <property name="text">
     <string>Atras</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_sagital">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Corte sagital&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_coronal">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Corte coronal&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_axial">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Corte axial&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="binarizacion">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>630</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;; background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>Binarización</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transformacion">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>680</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;;  background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>Tranformación</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bordes">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>680</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;;  background-color: rgb(9, 139, 148)</string>
    </property>
    <property name="text">
     <string>Bordes</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_procesada">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imagenes/img proy inf/truncfinal01.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="texto_img_procesada">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>570</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Imagen procesada&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo_pricipal.qrc"/>
 </resources>
 <connections/>
</ui>
